README.md

```diff
@@ -898,28 +897,32 @@ can be found in the [TraceTronic Knowledge Base](https://kb.tracetronic.com/disp
 - Jenkins LTS 2.319.3 or higher
 - Java SE Runtime Environment 11 or higher
  <details>
-    <summary><a href="https://www.tracetronic.com/products/ecu-test">ECU-TEST</a> 7.0.0 or higher </summary>
```

Comment:
Add untested versions information to the _Known limitations_ sections and refer to the compat matrix specification